lms/product/product.py

```diff
@@ -27,27 +28,50 @@ def _missing_(cls, _value):
 class Routes:
     """A collection of Pyramid route names for various functions."""
 
-    oauth2_authorize: str = None
+    oauth2_authorize: Optional[str] = None
     """Authorizing with OAuth 2."""
 
-    oauth2_refresh: str = None
+    oauth2_refresh: Optional[str] = None
     """Refreshing OAuth 2 tokens."""
 
+    list_group_sets: Optional[str] = None
+    """List available group sets. Takes a course_id parameter"""
+
+
+@dataclass
+class Settings:
+    """Product specific settings."""
+
+    product_settings: InitVar[Dict]
```

Comment:
`InitVar` can't have a `default_factory`

No need for a default in any case, just moved it to the top.